src/AWS.Deploy.Recipes/RecipeDefinitions/ConsoleAppECSFargateService.recipe

```diff
@@ -124,7 +124,7 @@
                     "Validators": [
                         {
                             "ValidatorType": "Regex",
-                            "Configuration" : {
+                            "Configuration": {
```

Comment:
nit: there's a lot of whitespace changes. Can you remove them?

Reply:
I think the VS did the autoformatting and it is making it the same as the other lines. So I think we should make the change to avoid getting this in the future.